op.js

```diff
@@ -105,28 +95,25 @@ const addDiffButtons = function () {
 	btnHolder.style.padding = '0';
 
 	// add buttons to buttons holder
-	var addButton = function (holder, text, handler) {
+	const addButton = function (holder, text, handler) {
 		const button = document.createElement('a');
 		button.classList.add('op-extensions-diff-button');
-		button.innerText = text
+		button.innerText = text;
 		button.addEventListener('click', handler);
 
 		holder.appendChild(button);
 	};
 
 
 	const setDisplayFor = function (selector, displayValue) {
-		const nodes = diffView.querySelectorAll(selector);
-		if (nodes) {
-			for (let i = 0; i < nodes.length; i++) {
-				nodes[i].style.display = displayValue;
-			}
-		}
+		[].forEach.call(diffView.querySelectorAll(selector), node => {
```

Comment:
Casting to the Array type just to be able to do the same with less lines of code?
I don't think that's efficient and necessary.

Reply:
ok, I will cancel this change